apps/web/src/stores/evaluationResultCharts/evaluationResultsCounters.ts

```diff
@@ -12,23 +14,24 @@ export default function useEvaluationResultsCounters(
     documentUuid: string
     evaluationId: number
   },
-  opts: SWRConfiguration = {},
+  { fallbackData }: SWRConfiguration = {},
 ) {
   const { project } = useCurrentProject()
+  const fetcher = useCallback(async () => {
+    const [data, error] = await computeEvaluationResultsCountersAction({
+      projectId: project.id,
+      commitUuid,
+      documentUuid,
+      evaluationId,
+    })
+
+    if (error) return null
```

Comment:
is the action returning an array? if so return an array also here so that the payload is always consistent